package.json

```
@@ -20,8 +20,8 @@
     "storybook:clean": "rm -rf storybook/public",
     "storybook:build": "storybook build -c storybook -o storybook/public --docs",
     "storybook:build-deps": "yarn workspace @kaizen/design-tokens prepublish && yarn workspace @kaizen/tailwind prepublish",
-    "playwright": "IS_DEV=true npx playwright test",
-    "playwright:debug": "IS_DEV=true PWDEBUG=1 npx playwright test",
+    "test:storybook": "STORIES=ignoreBrandMoment test-storybook --config-dir storybook",
```

```suggestion
    "test:storybook": "STORIES=whitelist test-storybook --config-dir storybook",
```

More generic name in case we need to ignore more?

---

Ideally we shouldn't be ignoring anything. I named it this to call out that it's super strange and ideally we'd fix it. It's actually the opposite of a whitelist hahaha.